devices/stm32h503.yaml

```
@@ -37,6 +37,23 @@ CRS:
 
 DAC:
   _strip: DAC_
+  _include: fields/dac/v6.yaml
+  MCR:
+    _merge: HFSEL?
+  CR:
+    TSEL?:
+      Software: [0, Software trigger]
+      Tim1Trgo: [1, Timer 1 TRGO event]
+      Tim2Trgo: [2, Timer 2 TRGO event]
+      Tim4Trgo: [3, Timer 4 TRGO event]
+      Tim5Trgo: [4, Timer 5 TRGO event]
+      Tim6Trgo: [5, Timer 6 TRGO event]
+      Tim7Trgo: [6, Timer 7 TRGO event]
+      Tim8Trgo: [7, Timer 8 TRGO event]
+      Tim15Trgo: [8, Timer 15 TRGO event]
```

Table 132 [RM0492](https://www.st.com/resource/en/reference_manual/rm0492-stm32h503-line-armbased-32bit-mcus-stmicroelectronics.pdf)

```suggestion
      Software: [0, Software trigger]
      Tim1Trgo: [1, Timer 1 TRGO event]
      Tim2Trgo: [2, Timer 2 TRGO event]
      Tim3Trgo: [3, Timer 3 TRGO event]
      Tim6Trgo: [5, Timer 6 TRGO event]
      Tim7Trgo: [6, Timer 7 TRGO event]
```